tests/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ include_directories(${Boost_INCLUDE_DIRS})
 set(HEADERS arms.h)
 
 macro(addUnitTest name)
-  if(${UNIT_TESTS})
+  if(NOT ${DISABLE_TESTS})
```

Comment:
This (and the related `endif` obviously) can be removed as the file will not be included if `DISABLE_TESTS` is ON.